orbital-mechanics/TheRealLecture1/SpacecraftDynamicsWeek1r;$.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>Here too.</p><p> </p><p>Hardit, how are you formatting your equations? Is it like Angadh showed in the closed Lecture 7 PR? Because they should all be numbered and you shouldn’t be needing dollar signs (if memory serves right)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>